### PR TITLE
fix format problems of kcs awards and some comments

### PR DESCRIPTION
**Pull request by pacoxu**
Update some wrong format and fix Peter Hunt and Manuel's award comments.
- a format error below: no new line after the name.

image

### CODE REVIEW

File: content/en/community/awards/2024/index.md [hunk 1]
```diff
@@ -129,26 +132,15 @@ Vyom has gone above and beyond in supporting the new Release Signal sub team ove
...
-
-*Rey Lejano, [@reylejano](https://github.com/reylejano)*  
-For the third-party audit, especially the post-audit follow-up, and so many other things across SIGs
-
-*Ashish Malik, [@ashish493](https://github.com/ashish493)*  
-For detailed and and collaborative work on the SIG Security Docs hardening guide
-
-*Kris Nóva, [@krisnova](https://github.com/krisnova)*  
-For so much Kubernetes security work, especially work that was done anonymously or was uncredited. We see you, we love you, and we miss you
-
```
Comment (pacoxu) [inline content/en/community/awards/2024/index.md:151]: This part is from 2023. 2024 SIG-Security awards is after SIG-Scheduling.

File: content/en/community/awards/2024/index.md [hunk 2]
```diff
@@ -114,10 +116,11 @@ Dave has been a prolific contributor to the Gateway API sub-project. Not only do
 #### Node
 
 *Peter Hunt, [@haircommander](https://github.com/haircommander)*  
-For the tireless and continuous pull request reviews and triage for SIG Node
+In recognition of your longstanding contributions and unwavering commitment to continuous improvement initiatives.
```
Comment (pacoxu) [inline content/en/community/awards/2024/index.md:119]: Find this comment in https://youtu.be/4WKTK0klhOI?t=448

**Comment by tokt** [comment]
/lgtm

Thank you so much for finding and fixing all these little inconsistencies, it's truly appreciated!

File: content/en/community/awards/2024/index.md [hunk 3]
```diff
@@ -51,39 +51,41 @@ For boosting so many messages both to and about our contributors, and taking on
...
-
 *Catherine Fang, [@catherineF-dev](https://github.com/CatherineF-dev)*  
 Catherine has easily been one of the most prominent contributors in Kube State Metrics since her involvement from the v2.6.0 release (August, 2022). Since then, she has not only unblocked us on various fronts by contributing crucial patches, but also been an excellent all-rounder, keeping the tooling and documentation in shape, as well as actively seeking out issues that affect the community, and addressing them timely. She has helped out with 204 pull request to date, while driving 34 pull requests of her own.
 
+*Manuel Rüger, [@mrueg](https://github.com/mrueg)*  
+Manuel has landed 144 pull requests to date, with patothes entailing major features, as well as keeping the release machineTV in shape which enabled him to deliver new releases ofn time.
+He has been active in the project since v1.5.0 (January, 2019), and been a major help ever since (involved in reviewing ~555 pull requests) Kube State Metrics could never have gotten to
```
Comment (jberkus) [inline content/en/community/awards/2024/index.md:86]: Kubernetes 1.5 was in 2016.  Is this supposed to say 2016, or is it supposed to say v1.14.0?

**Review by jberkus — CHANGES_REQUESTED** [review]
Thanks for finding the additional paste-oes.   Some additional corrections, though particularly to the description in Ruger's award.

**Review by sftim — COMMENTED** [review]
These changes make the page less wrong, so:
/approve

Further fixes are welcome!

File: content/en/community/awards/2024/index.md (hunk 3 above)
Comment (pacoxu) [reply content/en/community/awards/2024/index.md:86]: I think SIG-Instrumentation leads here v1.5 means https://github.com/kubernetes/kube-state-metrics/releases/tag/v1.5.0 😄.

**Pull request merged**